api/proto/buf.yaml

```diff
@@ -0,0 +1,29 @@
+version: v1
+deps:
+  - buf.build/gogo/protobuf:b03c65ea87cdc3521ede29f62fe3ce239267c1bc # v1.3.2
```

Comment:
Would it be worth it to put a comment here and a comment in the docker file where we install 1.3.2 in order to ensure they get updated together?

Reply:
Done.